drivers/syslog/Kconfig

```diff
@@ -215,20 +211,21 @@ config SYSLOG_RPMSG
 	---help---
 		Use the rpmsg as a SYSLOG output device, send message to remote proc.
 
-config SYSLOG_RPMSG_SERVER_NAME
-	string "The name of Syslog Rpmsg Server"
-	depends on SYSLOG_RPMSG
-	---help---
-		The proc name of rpmsg server. Client sends message to
-		specified name of remote proc.
-
 config SYSLOG_DEFAULT
 	bool "Default SYSLOG device"
+	default ARCH_LOWPUTC && !SYSLOG_RPMSG && !SYSLOG_CONSOLE && !RAMLOG_SYSLOG && !SYSLOG_CHAR
 	---help---
 		syslog() interfaces will be present, but all output will go to the
 		up_putc(ARCH_LOWPUTC == y) or bit-bucket(ARCH_LOWPUTC == n).
 
-endchoice
+endif
+
+config SYSLOG_RPMSG_SERVER_NAME
+	string "The name of Syslog Rpmsg Server"
+	depends on SYSLOG_RPMSG
```

Comment:
Missing default value for string config

Reply:
There isn't a suitable default value, so it's better to no default value at all and then enforce the user must give one in defconfig.

Reply:
Ok, I agree.
Does Kconfig assume `""` as the default value for a string config?
This enforcement is currently not implemented, right?

Reply:
Done!